frontend/src/screens/ApplicantOpenings.tsx

```diff
@@ -62,6 +66,15 @@ function RecruitmentRoundDetailsPage() {
   const [teamFilter, setTeamFilter] = useState("");
   const BASE_API_URL = getBaseAPIURL();
 
+  // sign out
+  const { signOut } = useAuthStore();
+  const navigate = useNavigate();
+
+  const handleSignOut = () => {
+    signOut();
+    navigate("/login");
+  };
+
```

Comment:
Why is this here?

Reply:
If it was because you cannot access login page while logged in, we will have to do another fix instead of logging user out